.github/workflows/macos-build.yml

```
@@ -0,0 +1,39 @@
+on:
+  push:
+    branches:
+      - main
+      - fix-mac-exe
```

Let's add testingv4artifact branch to the list of branches on which to run this workflow (for now), to test the workflow.
```suggestion
      - fix-mac-exe
      - testingv4artifact
```